examples/swc-plugin-exapmle/.gitignore

```diff
@@ -0,0 +1 @@
+.swc
```

Comment:
There's a typo in the directory name.

Reply:
This is the plugin cache file that swc will generate under the.swc directory
```
rspack_loader_swc
    .swc
        plugins
            v6_macos_aarch64_0.95.16
                757ef7d8d79efedecd0b28f05ecd66dc16126eb94c2fee5b8f97badcb7dea701
```

Reply:
What I mean is `swc-plugin-exapmle` -> `swc-plugin-example`

Reply:
> What I mean is `swc-plugin-exapmle` -> `swc-plugin-example`

@suxin2017 Would you fix this?